pkg/ttnpb/fieldmask_utils.go

```diff
@@ -182,10 +183,98 @@ outer:
 	return excluded
 }
 
+// EnsureFields appends all ensurePaths which are not already present in paths to paths and returns the result.
+func EnsureFields(paths []string, ensurePaths ...string) []string {
```

Comment:
Any better naming suggestion?
Is there an "upsert" analogue for appending?

Reply:
If we consider a FieldMask as a set type, Add would be fine too